docs/user/dql/functions.rst

```diff
@@ -1442,7 +1442,7 @@ DAYOFYEAR
 Description
 >>>>>>>>>>>
 
-Usage:  dayofyear(date) returns the day of the year for date, in the range 1 to 366.
+Usage:  dayofyear(date) returns the day of the year for date, in the range 1 to 366. Also supports day_of_year(date) as alternate syntax.
```

Comment:
Consider creating a section for `day_of_year` with one line description and reference to `dayofyear`. RST renderer adds an entry into table of content for each section automatically.